spec/spec.md

```diff
@@ -1379,9 +1379,11 @@ where [[ref:Verifiers]] and [[ref:Holders]] convey what [[ref:Claim]] format
 variants they can receive or are submitting. These formats are codified by
 abbreviation tags which are defined in a separate 
 [registry](https://identity.foundation/claim-format-registry/#registry) also 
-hosted at the DIF, and maintained by the same authors as this specification. To
-add abbreviations for a claim format supported by a working implementation, please
-follow [these instructions](https://identity.foundation/claim-format-registry/#structure-of-the-registry)
+hosted at the DIF, and maintained by the same authors as this specification. 
+The registry includes JSON Schema documents, which are referenced in schemas
+defined in this specification. To add abbreviations for a claim format supported
+ by a working implementation, please follow 
+ [these instructions](https://identity.foundation/claim-format-registry/#structure-of-the-registry)
```

Comment:
@bumblefudge does this cover https://github.com/decentralized-identity/presentation-exchange/issues/351?